cpg-analysis/build.gradle.kts

```diff
@@ -42,6 +42,7 @@ publishing {
 
 dependencies {
     api(projects.cpgCore)
+    api(projects.cpgLanguageJava)
```

Comment:
is that intentional? or do you only want to include it for the tests?

Reply:
It should only be required for the tests, so I changed it to `testImplementation()`.